### DISCUSSION TITLE
How to do callback for SFS?

### DISCUSSION THREAD

**Discussion by blazespinnaker** [rasbt/mlxtend#981] [Q&A]
SFS can take awhile sometimes, and it'd be useful to report on certain statistics in order to do early stopping or try a different experiment in a new window.   Is there a way to do a callback with the sequential feature selector at the end of each pass?

**Comment by rasbt** [comment]
Totally agree that this is useful. Currently, there is no way to do this, but it would be a cool feature to add.
